### ISSUE TITLE
Post series: Ability to subscribe to and receive notification when a new post is added

### ISSUE THREAD

**Issue by idris-rampurawala** [thepracticaldev/dev.to#6854]
**Is your feature request related to a problem? Please describe.**
No. It is not a problem but a powerful feature request.

**Describe the solution you'd like**
Dev recently introduced `Post Series` feature where we can post a series of articles and a link to all the posts in the series to go through. I would like to make this feature more powerful by adding a `Subscribe to series` functionality. When a person subscribed to any post series, he/she will get notified whenever a new post is added to that series. 

Currently, it is very difficult to keep a track of post series for the reader. This feature will certainly help the reader more flexibility in terms of being notified for his/her interests.

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
@idris-rampurawala Thank you for the suggestion. In the meantime, you can follow the author of an article series to be notified about all of their new posts. I know this is not ideal since you may end up with more notifications than intended, but maybe it will help you not to miss new posts in series you follow.

**Comment by rhymes** [comment]
Thanks @idris-rampurawala, I'm closing this as a duplicate of https://github.com/thepracticaldev/dev.to/issues/1172 as I think they are the same issue, let me know otherwise!
